code/modules/surgery/bodyparts/parts.dm

```diff
@@ -70,30 +70,41 @@
 	bodytype = BODYTYPE_LARVA_PLACEHOLDER | BODYTYPE_ORGANIC
 	acceptable_bodytype = BODYTYPE_LARVA_PLACEHOLDER
 
-/obj/item/bodypart/l_arm
+/obj/item/bodypart/arm
+	name = "arm"
+	desc = "Need a hand?"
+	attack_verb_continuous = list("slaps", "punches")
+	attack_verb_simple = list("slap", "punch")
+	max_damage = 50
+	max_stamina_damage = 50
+	aux_layer = HANDS_PART_LAYER
+	body_damage_coeff = 0.75
+	can_be_disabled = TRUE
+	unarmed_attack_verb = "punch" /// The classic punch, wonderfully classic and completely random
+	unarmed_damage_low = 1
+	unarmed_damage_high = 10
+	unarmed_stun_threshold = 10
+	///If our fingers are too big to fire guns
+	var/chunky = FALSE
```

Comment:
I don't like this being hard coded as a var on bodyparts.

Reply:
Do you have an alternative method? It would need to be limb specific. The only thing I can think of would be a trait on the limb, I could implement a system to give limbs traits but that would also require a var to tell them what traits to give.

Reply:
limb_traits would be good.